funcx_endpoint/funcx_endpoint/endpoint/endpoint.py

```diff
@@ -145,6 +145,7 @@ def init_endpoint():
     init_endpoint_dir("default")
 
 
+@retry(delay=10, logger=logging.getLogger('funcx'))
```

Comment:
This is nice. It might be useful to add on a couple of additional options here like `backoff=1.2, max_delay=300`.

Reply:
Updated.